Tests/Fixtures/Form/Guesser.php

```diff
@@ -35,18 +35,12 @@ class Guesser
     /** @ODM\Field(type="bool") */
     public $boolField;
 
-    /** @ODM\Field(type="boolean") */
```

Comment:
I had this error in phpunit
```
Remaining indirect deprecation notices (2)

  1x: Since doctrine/mongodb-odm 2.1: The "boolean" mapping type is deprecated. Use "bool" instead.
    1x in TypeGuesserTest::testTypesShouldBeGuessedCorrectly from Doctrine\Bundle\MongoDBBundle\Tests\Form\Type

  1x: Since doctrine/mongodb-odm 2.1: The "integer" mapping type is deprecated. Use "int" instead.
    1x in TypeGuesserTest::testTypesShouldBeGuessedCorrectly from Doctrine\Bundle\MongoDBBundle\Tests\Form\Type
```

Reply:
It'd be the best to extract these two asserts into a separate test that would allow deprecation notices but considering simplicity of the guesser I think we'll be just fine